main/pong.go

```diff
@@ -0,0 +1,587 @@
+/*
+	pong.go: uAvionix Pong ADS-B monitoring and management.
+	Added 1/2026
+*/
+
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+	"sync"
+
+	//"sync/atomic"
+	"net"
+	"os/exec"
+	"time"
+
+	// Using forked version of tarm/serial to force Linux
+	// instead of posix code, allowing for higher baud rates
+	"github.com/stratux/serial"
+	"github.com/stratux/stratux/common"
+)
+
+// pong device data
+var pongSerialConfig *serial.Config
+var pongSerialPort *serial.Port
+var pongWG *sync.WaitGroup
+var closeChpong chan int
+var pongUpdateMode bool
+var pongDeviceSuccessfullyWorking bool
+
+
+type PongTermMessage struct {
+	Text   string
+	Source string
+}
+
+func initPongSerial() bool {
+	var device string
+	baudrate := int(3000000)
+
+	log.Printf("Configuring Pong ADS-B\n")
+
+	if _, err := os.Stat("/dev/pong"); err == nil {
+		device = "/dev/pong"
+	} else {
+		log.Printf("No suitable Pong device found.\n")
+		return false
+	}
+	log.Printf("Using %s for Pong\n", device)
+
+	// Open port
+	// No timeout specified as Pong does not heartbeat
+	pongSerialConfig = &serial.Config{Name: device, Baud: baudrate}
+	p, err := serial.OpenPort(pongSerialConfig)
+	if err != nil {
+		log.Printf("Error opening serial port: %s\n", err.Error())
+		return false
+	}
+	errRts := p.ClearRTS()
+	if errRts != nil {
+		log.Printf("Clearing RTS returned %s\n", errRts.Error())
+	}
+
+	log.Printf("Pong opened serial port at %d baud\n",baudrate)
+
+	// No device configuration is needed, we should be ready
+	globalStatus.Pong_Heartbeats = 0
+
+	pongSerialPort = p
+	return true
+}
+
+func pongNetworkRepeater() {
+	defer pongWG.Done()
+	log.Println("Entered Pong network repeater ...")
+	cmd := exec.Command(STRATUX_HOME + "/bin/dump1090", "--net-only", "--net-stratux-port", "30006")
```

Comment:
Hmm, port 30006 here seems to be in conflict with the normal Stratux dump1090 instance.
I'd guess that if you have a ping or a pong, you don't really care about having and SDR for 1090, too, so probably fine.